src/GraphQL/Resolver/DomainContentResolver.php

```diff
@@ -72,12 +72,15 @@ public function resolveDomainContentItems($contentTypeIdentifier, $query = null)
     /**
      * Resolves a domain content item by id, and checks that it is of the requested type.
      *
-     * @param \Overblog\GraphQLBundle\Definition\Argument $args
-     * @param $contentTypeIdentifier
+     * @param \Overblog\GraphQLBundle\Definition\Argument|array $args
+     * @param string|null $contentTypeIdentifier
      *
      * @return \eZ\Publish\API\Repository\Values\Content\Content
+     *
+     * @throws \GraphQL\Error\UserError if $contentTypeIdentifier was specified, and the loaded item's type didn't match it
+     * @throws \GraphQL\Error\UserError if no argument was provided
      */
-    public function resolveDomainContentItem(Argument $args, $contentTypeIdentifier)
+    public function resolveDomainContentItem($args, $contentTypeIdentifier)
```

Comment:
Did it change in overblog graphql package? I though it would always return an `Argument` object which implements `ArrayAccess` if I'm not mistaken.

Reply:
I did change this because in this PR, to resolve a Location's `content` field, we call the method directly, with an array as the argument. In `src/Resources/config/graphql/Location.types.yml`: `resolve: "@=resolver('DomainContentItem', [{id: value.contentId}, null])"`.

I agree that it's not the cleanest thing ever, but... 😊